packages/components/src/utils/config-values.js

```diff
@@ -58,7 +58,7 @@ export default Object.assign( {}, CONTROL_PROPS, TOGGLE_GROUP_CONTROL_PROPS, {
 	fontSizeMobile: '15px',
 	fontSizeSmall: 'calc(0.92 * 13px)',
 	fontSizeXSmall: 'calc(0.75 * 13px)',
-	fontLineHeightBase: '1.2',
+	fontLineHeightBase: '1.4',
```

Comment:
This feels like it could have significant impact.

Reply:
Why not add this as a specific css for the specific `ConfirmDialog` we're having a problem with?

Reply:
I could do but then it feels like patching over a problem instead of dealing with the root cause.